themes/src/main/resources/theme/base/account/resources.ftl

```diff
@@ -316,11 +316,8 @@
         </div>
         <div class="row">
             <div class="col-md-12">
-                <#if authorization.resourcesWaitingOthersApproval?size != 0>
```

Comment:
This `if` statement is not needed because the same line exists in line 308:
https://github.com/keycloak/keycloak/pull/6149/files#diff-f9cab8b2b8715c8860a52ab818f3fdadL308